seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java

```diff
@@ -139,7 +130,7 @@ private SeaTunnelRowType initTableField(Connection conn) {
         ArrayList<String> fieldNames = new ArrayList<>();
         try {
             PreparedStatement ps = conn.prepareStatement(jdbcSourceOptions.getJdbcConnectionOptions().getQuery());
-            ResultSetMetaData resultSetMetaData = ps.getMetaData();
+            ResultSetMetaData resultSetMetaData = ps.executeQuery().getMetaData();
```

Comment:
why need add executeQuery

Reply:
executeQuery has a great impact on the performance of other databases, it is recommended to use JdbcDialect to achieve